styles/layout.css

```diff
@@ -0,0 +1,9 @@
+@media only and (min-width: 640px) {
+	.nav ul {
+		display: block;
+	}
+}
```

Comment:
You always want your @media queries to be below your default CSS. As it is, your default CSS is overriding any rules in your @media query.